ext/ext_helper.rb

```diff
@@ -0,0 +1,23 @@
+require 'fileutils'
+
+module Platform
+   def self.architecture
+      ['foo'].pack('p').size == 8 ? 'x64' : 'x86'
+   end
+end
+
+module ExternalBinaries
+   extend self
+
+   def move_files(externals)
```

Comment:
Since this is only used by gemspec and it does not have any external dependencies then maybe move this function into gemspec local function instead?

Reply:
I actually started out trying to do it like this, but I kept getting a 'undefined method' error when doing so. I have tried looking around in other gemspecs to see if anyone has a local function defined but I haven't had any luck. Do you have any experience in doing it this way?